library/tasks/RunEmptyDrops.wdl

```diff
@@ -0,0 +1,47 @@
+task RunEmptyDrops {
+    # Input data
+    File sparse_count_matrix
+    File col_index
+    File row_index
+
+    # emptyDrops Params
+    Float niters = 10000
+    Float fdr_cutoff = 0.10
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-dropletutils:test"
```

Comment:
Reminder: please remember to change this to something like: `String docker = "quay.io/humancellatlas/secondary-analysis-dropletutils:v0.2.0"` before merging

Reply:
Will do (I'll also test manually before merge after the tag update)

Reply:
👍